#### A statistical analysis that combines information from several studies is called a meta analysis. A meta analysis compared aspirin with placebo on incidence of heart attack and of stroke, separately for men and for women (J. Am. Med. Assoc., 295: 306–313, 2006). For the Women’s Health Study,heart attacks were reported for 198 of 19,934 taking aspirin and for 193 of 19,942 taking placebo.

###### a. Construct the 2 × 2 table that cross classifies the treatment (aspirin,placebo) with whether a heart attack was reported (yes, no).

In [2]:
# 2x2 table
b1<-198
b2<- 19934-b1
w1<- 193
w2<-19942-w1
CTable_12<-array(data=c(b1,w1,b2,w2),dim=c(2,2),
dimnames=list(Treatment=c('Aspirin','Placebo'),
Reported=c("Y","N")))
CTable_12


# 2x2 table (probability)
pi.hat.table.12<-CTable_12/rowSums(CTable_12)
pi.hat.table.12

Reported
Treatment Y   N    
  Aspirin 198 19736
  Placebo 193 19749

Reported
Treatment Y           N        
  Aspirin 0.009932778 0.9900672
  Placebo 0.009678066 0.9903219

###### b. Estimate the odds ratio. Interpret.
Odds Ratio Interpretation: The odds of a women getting a heart attack when they're on aspirin are 1.03 times higher than when they're on the placebo. 

In [4]:
OR.hat12<-CTable_12[1,1]*CTable_12[2,2] / (CTable_12[2,1]*CTable_12[1,2]) #(a/b)/(c/d)
OR.hat12

#Double Checking
A<-0.009932778/0.9900672
P<-0.009678066/0.9903219
A>P
A/P

[1] 1.026582

[1] TRUE

[1] 1.026583

###### c. Find a 95% confidence interval for the population odds ratio for women.Interpret. (As of 2006, results suggested that for women, aspirin was helpful for reducing risk of stroke but not necessarily risk of heart attack.)

In [5]:
alpha<-0.05
var.log.or<-1/CTable_12[1,1] + 1/CTable_12[1,2] + 1/CTable_12[2,2] + 1/CTable_12[2,1]
OR.CI<-exp(log(OR.hat12) + qnorm(p = c(alpha/2, 1-alpha/2)) * sqrt(var.log.or))
round(OR.CI, 4)

[1] 0.8411 1.2529

The 95% confidence interval for the population odds ratio for women is (0.8411,1.2529).
We are 95% confident that that the population odds ratio for women is between 0.8411 and 1.2529
Because 1 is within the interval, we have evidence that the true odds for the two groups are the same; therefore, the odds are not dependent on the group. 